src/gui/screen_menu_tune.cpp

```diff
@@ -23,6 +17,7 @@ enum {
     MI_FLOWFACT,
     MI_BABYSTEP,
     MI_FILAMENT,
+    MI_INFO,
```

Comment:
Why is the `INFO` screen back? Isn't it a mistake?

Reply:
why was it deleted?
i thought was deleted by a mistake

Reply:
removed